tools/ci/cirrus_wheels.yml

```
@@ -55,112 +16,34 @@ linux_aarch64_task:
 macosx_arm64_task:
   use_compute_credits: $CIRRUS_USER_COLLABORATOR == 'true'
   macos_instance:
-    image: ghcr.io/cirruslabs/macos-monterey-xcode:14
+    matrix:
+      - image: ghcr.io/cirruslabs/macos-monterey-xcode:14
+      #- image: ghcr.io/cirruslabs/macos-sonoma-xcode:latest
   matrix:
     - env:
         CIRRUS_CLONE_SUBMODULES: true
-        CIBW_BUILD: cp39-*
-    - env:
-        CIRRUS_CLONE_SUBMODULES: true
-        CIBW_BUILD: cp310-* cp311-*
-    - env:
-        CIRRUS_CLONE_SUBMODULES: true
-        CIBW_PRERELEASE_PYTHONS: True
-        CIBW_BUILD: cp312-*
+        CIBW_BUILD: cp310-*
+        # Specifying CIBW_ENVIRONMENT_MACOS overrides pyproject.toml, so include
+        # all the settings from there, otherwise they're lost.
   env:
     PATH: /opt/homebrew/opt/python@3.10/bin:/usr/local/lib:/usr/local/include:$PATH
     CIBW_ARCHS: arm64
-    # Specifying CIBW_ENVIRONMENT_MACOS overrides pyproject.toml, so include
-    # all the settings from there, otherwise they're lost.
-    # SDKROOT needs to be set for repackaged conda-forge gfortran compilers
-    # supplied by isuruf.
-    # Find out SDKROOT via `xcrun --sdk macosx --show-sdk-path`
     CIBW_ENVIRONMENT_MACOS: >
       RUNNER_OS=macOS
+      MACOSX_DEPLOYMENT_TARGET=11.0  # 14.0
+      # SDKROOT needs to be set also here (see gfortran_utils.sh)
       SDKROOT=/Applications/Xcode-14.0.0.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX12.3.sdk
```

This path might be wrong.
I suggest
```suggestion
      SDKROOT=/Applications/Xcode-14.0.0.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk
```

---

The test run tries to compile a fortran program in a subprocess. I suspect that this env variable isn't being passed down to the compilation.